Doc/howto/regex.rst

```diff
@@ -549,27 +545,39 @@ of each one.
 
    Perform case-insensitive matching; character class and literal strings will
    match letters by ignoring case.  For example, ``[A-Z]`` will match lowercase
-   letters, too, and ``Spam`` will match ``Spam``, ``spam``, or ``spAM``. This
-   lowercasing doesn't take the current locale into account; it will if you also
-   set the :const:`LOCALE` flag.
+   letters, too. Full Unicode matching also works unless the :const:`re.ASCII`
+   flag is also used to disable non-ASCII matches.  ``[A-Z]`` will also match
+   letters 'İ' (U+0130, Latin capital letter I with dot above), 'ı' (U+0131,
+   Latin small letter dotless i), 'ſ' (U+017f, Latin small letter long s) and
+   'K' (U+212a, Kelvin sign) in Unicode mode.  ``Spam`` will match ``Spam``,
```

Comment:
shouldn't all those after "will match" be ``'[word]'`` instead of ``[word]``?

Reply:
``Spam``, ``spam``, ``spAM``, or ``ſpam`` ->  ``'Spam'``, ``'spam'``, ``'spAM'``, or ``'ſpam'``? I might have gotten it wrong though.

Reply:
I think he's referring to the 4 non-ASCII letters: they should be enclosed in  `` `'..'` ``